README.md

```diff
@@ -1,23 +1,23 @@
 # numericycle
```

Comment:
Can you include this code in your README?
`[![Build Status](https://travis-ci.com/DEGJS/numericycle.svg?branch=master)](https://travis-ci.com/DEGJS/numericycle)`

It's for the Travis build badge